fw/cache.c

```diff
@@ -2628,7 +2623,7 @@ tfw_cache_add(TfwHttpResp *resp, tfw_http_cache_cb_t action)
 		 * rather than in softirq...
 		 */
 		for_each_node_with_cpus(nid)
-			__cache_add_node(c_nodes[nid].db, resp, key);
+			__cache_add_node(get_db_for_node(nid), resp, key);
```

Comment:
Here we doing `__cache_add_node()`, however more proper way to do it is to call only `tfw_cache_copy_resp()` for each node, instead of calling whole `__cache_add_node()`. I would like to do it in another patch if we not decide to move NUMA to TDB.